api/package.json

```diff
@@ -73,7 +73,7 @@
     "@typescript-eslint/parser": "~3.7.1",
     "chai": "~4.2.0",
     "del": "~6.0.0",
-    "eslint": "~7.5.0",
+    "eslint": "^7.5.0",
```

Comment:
should this not be changed?

Reply:
didn't make this change so not sure of the impact

Reply:
<img width="474" alt="Screenshot 2021-03-11 093329" src="https://user-images.githubusercontent.com/28017034/110829431-f5ece300-824c-11eb-9a6d-fa9eadf6e803.png">

I think we wanna keep it as a `~` not a `^`